perfkitbenchmarker/gcp/gce_virtual_machine.py

```diff
@@ -101,6 +102,8 @@ def _Create(self):
                     'sshKeys=%s' % tf.name,
                     '--metadata',
                     'owner=%s' % FLAGS.owner]
+      for key, value in self.boot_metadata.iteritems():
+        create_cmd.append('%s=%s' % (key, value))
       ssd_interface_option = NVME if NVME in self.image else SCSI
```

Comment:
Right - but the ssd_interface_option will only be "nvme" if the string "nvme" is present in the image name - which it isn't for windows. So this works as it's supposed to